core/src/main/java/org/apache/carbondata/core/datastore/chunk/reader/measure/v3/CompressedMeasureChunkFileBasedReaderV3.java

```diff
@@ -220,13 +223,25 @@ public CompressedMeasureChunkFileBasedReaderV3(BlockletInfo blockletInfo, String
       valueEncodeMeta.add(CarbonUtil
           .deserializeEncoderMetaNew(measureColumnChunk.getEncoder_meta().get(i).array()));
     }
-    WriterCompressModel compressionModel = CarbonUtil.getValueCompressionModel(valueEncodeMeta);
-    ValueCompressionHolder values = compressionModel.getValueCompressionHolder()[0];
+
+    MeasurePageStatistics stats = CarbonUtil.getMeasurePageStats(valueEncodeMeta);
```

Comment:
This code is repeated in CompressedMeasureChunkFileBasedReaderV2 and CompressedMeasureChunkFileBasedReaderV3, if we move it to a shared function, a result object need to be created, since multiple results are needed (stats, convertedType, values). 

Actually in #987 , usage of `CompressionFinder` is removed in write path, I think another PR is needed to remove it for read path also, after this is done, `CompressionFinder` related class and the whole `org.apache.carbondata.core.datastore.compression` package.